Tests/ValetIntegrationTests/SinglePromptSecureEnclaveIntegrationTests.swift

```diff
@@ -35,9 +35,6 @@ class SinglePromptSecureEnclaveIntegrationTests: XCTestCase
     {
         super.setUp()
 
-        guard #available(tvOS 11.0, *) else {
-            return
-        }
```

Comment:
"Fun" fact: Marking a test as `@available` does not prevent that test from being run on unavailable OS versions 😬, hence the need for all of these `guard`s

Reply:
I figure that out, hehe